Add "Tile usage" notice at the top of doc page for OSM maps and geocoding (#11411)
decidim/decidim: docs/modules/services/pages/maps.adoc

= Maps and geocoding

Decidim has the ability to geocode proposals and meetings and display them on a map.

[CAUTION]
====

> It is important to notice that you can not use the OpenStreetMap service from openstreetmap.org. As they mention in their https://operations.osmfoundation.org/policies/tiles/[tile usage policy]:

Reviewer: ```suggestion
It is important to notice that you cannot use the OpenStreetMap service from openstreetmap.org. As they mention in their https://operations.osmfoundation.org/policies/tiles/[tile usage policy]:
```
